.github/workflows/check.yaml

```diff
@@ -10,9 +10,11 @@ on:
       - ready_for_review
     branches:
       - main
+      - filter_panel_refactor@main
```

Comment:
why are you changing these values? I think it should be first changed in filter_panel_refactor@main and merged in to your branch.

Reply:
I did not change it manually.  at last commit  [abaeaf2](https://github.com/insightsengineering/teal.slice/pull/296/commits/abaeaf252ee8dd98ea8a11c51a608f17a92ae7b6) this got updated.